src/index.html

```diff
@@ -5,6 +5,7 @@
   <meta http-equiv="x-ua-compatible" content="ie=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,800" rel="stylesheet">
+  <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" rel="stylesheet">
```

Comment:
Hmmm, correct. I don't think this is the best way to accomplish this. I thought the ng2-bootstrap components contained their own styles? 

If this is included to get the list displaying properly, that's something that could better be handled by flexbox and the existing classes in the project I think. @jfrankl do you have thoughts on this?